test/metabase/upload_test.clj

```diff
@@ -384,11 +388,13 @@
                 (is (not= (:id table-1)
                           (:id table-2)))))))))))
 
-(defn- query-table
-  [table]
-  (qp/process-query {:database (:db_id table)
+(defn- query [db-id source-table]
```

Comment:
with dynamic typing it always pays to be consistent with names and types, so I'd suggest renaming this
```suggestion
(defn- query [db-id source-table-id]
```

Reply:
This argument stumped me a bit, because it could either be a numeric table id or string card reference. I felt like playing it safe so used the same name as the key it's assigned to.